### PR TITLE
[pkp-lib] #3080 CurrentContext not available on Site level fix stable

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl [hunk 1]
```diff
@@ -48,7 +48,11 @@
 				{fbvFormSection title="manager.navigationMenus.form.path" for="path" required="true"}
 					{fbvElement type="text" id="path" value=$path required="true"}
 					<p>
-						{url|replace:"REPLACEME":"%PATH%"|assign:"exampleUrl" router=$smarty.const.ROUTE_PAGE context=$currentContext->getPath() page="navigationMenu" op="view" path="REPLACEME"}
+                        {if $currentContext}
```
Comment (asmecher) [inline templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl]: Space-based indents. Please config your editor to use tabs, and search/replace for these.

File: templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl [hunk 2]
```diff
@@ -48,7 +48,11 @@
...
+                            {url|replace:"REPLACEME":"%PATH%"|assign:"exampleUrl" router=$smarty.const.ROUTE_PAGE context=$currentContext->getPath() page="navigationMenu" op="view" path="REPLACEME"}
```
Comment (asmecher) [inline templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl]: Are you sure this is necessary? When `context` is not specified, the current context's path should be used already.

Comment (asmecher) [reply templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl]: (I think an equivalent but shorter fix would just be to remove the `context=...` parameter from the `{url ...}` call that was originally here.)

Comment (defstat) [reply templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl]: You are right @asmecher ... I changed it following your suggestion.

**Comment by asmecher** [comment]
Merged and ported to `master`, thanks!

**Pull request merged**